src/parser/jobs/brd/modules/RagingStrikes.tsx

```diff
@@ -218,4 +309,27 @@ export class RagingStrikes extends BuffWindow {
 		const firstGcd = gcdTimestamps[0]
 		return this.barrageRemoves.some(timestamp => firstGcd <= timestamp && timestamp <= (window.end ?? window.start))
 	}
+
+	private wasActionUsedInTime = (action: ActionKey, bestBeforeGCD: number, window: HistoryEntry<EvaluatedAction[]>): boolean => {
+		const buffAction = _.first(window.data.filter(it => it.action.id === this.data.actions[action].id))
+
+		if (buffAction) {
```

Comment:
We've shifted to using explicit checks for undefined values recently rather than relying on their falsiness. Something like this:

```suggestion
		if (buffAction != null) {
```